plugins/catalog-backend/src/service/DefaultEntitiesCatalog.ts

```diff
@@ -295,4 +298,49 @@ export class DefaultEntitiesCatalog implements EntitiesCatalog {
       items,
     };
   }
+
+  async facets(request: EntityFacetsRequest): Promise<EntityFacetsResponse> {
+    const { entities } = await this.entities({
+      filter: request.filter,
+      authorizationToken: request.authorizationToken,
+    });
+
+    const facets: EntityFacetsResponse['facets'] = {};
+
+    for (const facet of request.facets) {
+      const values = entities
+        .map(entity => {
+          // TODO(freben): Generalize this code to handle any field that may
+          // have dots in its key?
```

Comment:
:doit:

Reply:
but i am le tired

Reply:
To be a bit more clear - we don't allow dots anywhere else and I thought I'd just leave this unaddressed for now :)